src/components/DropdownList.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import debounce from 'lodash.debounce';
+import cn from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  onSelect: (person: Person | null) => void;
+  people: Person[];
+  delay?: number
+};
+
+export const DropdownList: React.FC<Props> = ({ onSelect, people, delay }) => {
+  const [query, setQuery] = useState<string>('');
+  const [appliedQuery, setAppliedQuery] = useState<string>('');
+  const [isShown, setIsShown] = useState<boolean>(false);
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
  const [appliedQuery, setAppliedQuery] = useState('');
  const [isShown, setIsShown] = useState(false);
```
it's redundant to specify the primitive types